tests/integration/aof.tcl

```diff
@@ -511,4 +511,23 @@ tags {"aof external:skip"} {
         }
     }
 
+    test {EVAL can't processes writes in AOF when doing "debug loadaof" in read-only slaves} {
```

Comment:
I don't think we wanna mention `debug loadaof` in the test.
the test comes to test aof loading on a replica, instead of doing it properly, it used the debug command (as a way to achieve something similar), but that's not the purpose of the test.

Actually, now that we generate the AOF on our own, we can just let the server start from it, and there's no need for DEBUG LOADAOF at all.

I also don't understand why you negated the name of the test (adding "can't") and why the assert is for "" and not "102"?

maybe there's a bug to fix?

```suggestion
    test {EVAL processes writes from AOF in read-only replicas} {
```

Reply:
ohh, i now see your other post, i suppose this is a bug we need to fix.
but either way, this test should now use plain AOF loading (start a server to be configured as a read-only replica, and set it to load an AOF file on startup)